src/app/screens/connect/authenticationRequest/index.tsx

```diff
@@ -214,11 +301,47 @@ function AuthenticationRequest() {
       loading={loading}
       disabled={isDisabled}
     >
-      <AccountHeaderComponent />
       <MainContainer>
-        <TopImage src={getDappLogo()} alt="Dapp Logo" />
+        {getDappLogo()}
         <FunctionTitle>{t('TITLE')}</FunctionTitle>
         <DappTitle>{`${t('REQUEST_TOOLTIP')} ${authRequest.payload.appDetails?.name}`}</DappTitle>
+        <SelectAccount account={selectedAccount!} handlePressAccount={handleSwitchAccount} />
+        <AddressesContainer>
+          <AddressBox>
+            <AddressContainer>
+              <AddressImage src={stxIcon} />
+              <AddressTextTitle>{t('STX_ADDRESS')}</AddressTextTitle>
+            </AddressContainer>
+            <div>
+              {selectedAccount?.bnsName ? <BnsName>{selectedAccount?.bnsName}</BnsName> : null}
+              <TruncatedAddress>
+                {getTruncatedAddress(selectedAccount?.stxAddress!)}
+              </TruncatedAddress>
+            </div>
+          </AddressBox>
+          <AddressBox>
+            <AddressContainer>
+              <AddressImage src={BitcoinIcon} />
+              <AddressTextTitle>{t('BITCOIN_ADDRESS')}</AddressTextTitle>
+            </AddressContainer>
+            <TruncatedAddress>{getTruncatedAddress(selectedAccount?.btcAddress!)}</TruncatedAddress>
+          </AddressBox>
```

Comment:
this should reuse a shared component with btcSelectAddressScreen, same as what you did for selectAccount